buffalo/cmd/soda_generators.go

```diff
@@ -19,11 +20,25 @@ func newSodaGenerator() *gentronics.Generator {
 	f.Should = should
 	g.Add(f)
 
-	c := gentronics.NewCommand(goGet("github.com/markbates/pop/..."))
-	c.Should = should
-	g.Add(c)
+	g.Add(&gentronics.Func{
+		Should: should,
+		Runner: func(rootPath string, data gentronics.Data) error {
+			var driver string
+			if data["dbType"] == "sqlite3" {
```

Comment:
> just a thought, but if we drop the _driver from the build tag then we don't need this if statement.
Sure. I've dropped the suffix on the pop side, so I've just to adjust the buffalo side now.

> also, it might be possible that people have two different DBs, say SQLite in development or test and Postgres in production. That's more of a thing we need to worry about during buffalo build. For that we can look at the database.yml file during build time and grab all of the dialects being used there to pass as flags.

Hum, I already thought about reading the `database.yml` for the build part, I'm glad you thought about it too. In this case, maybe we can provide the whole drivers set for the development part, and take care of required drivers only on build time? (and in this case, I can add a special `all` flag, to provide all drivers support, without having to enumerate them all)